gunpowder/torch/nodes/train.py

```diff
@@ -315,6 +315,20 @@ def train_step(self, batch, request):
 
         if self.summary_writer and batch.iteration % self.log_every == 0:
             self.summary_writer.add_scalar("loss", batch.loss, batch.iteration)
+            if hasattr(self.loss, 'loss_dict'):
```

Comment:
This section doesn't seem to be relevant to adding Resample/AsType

Reply:
Correct. I will remove some commits that were accidentally pushed to this PR. Thanks!

Reply:
Fixed with commits: 
[e2d09e6](https://github.com/funkey/gunpowder/pull/163/commits/e2d09e6ee19f2481729c1b625c1b0b3cb472e97c)
[dac8d27](https://github.com/funkey/gunpowder/pull/163/commits/dac8d27ed2adea09637bc8819fc1364e9861f0ce)